dataclasses_avroschema/model_generator/generator.py

```diff
@@ -118,9 +130,14 @@ def render_class(self, *, schema: JsonDict) -> str:
         rendered_fields = self.field_identation.join(
             [self.render_field(field=field, model_name=name) for field in record_fields]
         )
+        docstring = self.render_docstring(docstring=schema.get("doc"))
 
         rendered_class = templates.class_template.safe_substitute(
-            name=name, decorator=self.base_class_decotator, base_class=self.base_class, fields=rendered_fields
+            name=name,
+            decorator=self.base_class_decotator,
+            base_class=self.base_class,
+            fields=rendered_fields,
+            docstring=docstring,
```

Comment:
This will cause that the documentation will be generated in 2 places: `in classdocstring` and in the`schema_dod Class Meta`. I think we should use one, in this case the `classdocstring` seems a better option